plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/sources/SingleKeyNetworkTableSource.java

```diff
@@ -21,7 +22,7 @@
    * @param dataType the allowable data type. A value that is not an instance of this
    *                 type is considered "null" and will make the source inactive
    */
-  public SingleKeyNetworkTableSource(ITable table, String key, DataType dataType) {
+  public SingleKeyNetworkTableSource(NetworkTable table, String key, DataType dataType) {
     super(key, dataType);
     setName(key);
     setTableListener((__, value, flags) -> {
```

Comment:
I'm not sure what setTableListener does, but since you're listening to only a single key, I think you should use NetworkTableEntry.addListener() instead.